content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -132,13 +159,28 @@ While Kubernetes does not provide a native solution for cluster-level logging, t
 
 ![Using a node level logging agent](/images/docs/user-guide/logging/logging-with-node-agent.png)
 
-You can implement cluster-level logging by including a _node-level logging agent_ on each node. The logging agent is a dedicated tool that exposes logs or pushes logs to a backend. Commonly, the logging agent is a container that has access to a directory with log files from all of the application containers on that node.
-
-Because the logging agent must run on every node, it's common to implement it as either a DaemonSet replica, a manifest pod, or a dedicated native process on the node. However the latter two approaches are deprecated and highly discouraged.
-
-Using a node-level logging agent is the most common and encouraged approach for a Kubernetes cluster, because it creates only one agent per node, and it doesn't require any changes to the applications running on the node. However, node-level logging _only works for applications' standard output and standard error_.
-
-Kubernetes doesn't specify a logging agent, but two optional logging agents are packaged with the Kubernetes release: [Stackdriver Logging](/docs/user-guide/logging/stackdriver) for use with Google Cloud Platform, and [Elasticsearch](/docs/user-guide/logging/elasticsearch). You can find more information and instructions in the dedicated documents. Both use [fluentd](http://www.fluentd.org/) with custom configuration as an agent on the node.
+You can implement cluster-level logging by including a _node-level logging 
+agent_ on each node. The logging agent is a dedicated tool that exposes logs or 
+pushes logs to a backend. Commonly, the logging agent is a container that has 
+access to a directory with log files from all of the application containers on 
+that node.
+
+Because the logging agent must run on every node, it's common to implement it 
+as either a DaemonSet replica, a manifest pod, or a dedicated native process on 
+the node. However the latter two approaches are deprecated and highly 
+discouraged.
+
+Using a node-level logging agent is the most common and encouraged approach for 
+a Kubernetes cluster, because it creates only one agent per node, and it 
+doesn't require any changes to the applications running on the node. However, 
+node-level logging _only works for applications' standard output and standard 
+error_.
+
+Kubernetes doesn't specify a logging agent, and no logging agents are
+packaged with the Kubernetes core release. Third-party logging agents are
+beyond the scope of the Kubernetes documentation. The [CNCF Cloud Native
```

Comment:
Remove:
> Third-party logging agents are
beyond the scope of the Kubernetes documentation. 

It's true, but it's not immediately helpful in this context.